### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by vardan018** [vardan018/DL-project#1]
ILSVRC/ contains the image data and ground truth for the train and validation sets, and the image data for the test set.
The image annotations are saved in XML files in [PASCAL VOC format](http://host.robots.ox.ac.uk/pascal/VOC/voc2008/htmldoc/voc.html#SECTION00031000000000000000). Users can parse the annotations using the PASCAL Development Toolkit.
Annotations are ordered by their synsets (for example, "Persian cat", "mountain bike", or "hot dog") as their wnid. These id's look like n00141669. Each image's name has direct correspondence with the annotation file name. For example, the bounding box for n02123394/n02123394_28.xml is n02123394_28.JPEG.
Bounding boxes of a particular synset from http://www.image-net.org/api/download/imagenet.bbox.synset?wnid=[wnid]
The training images are under the folders with the names of their synsets.  The test images are also all in the same folder.

Link to the dataset: https://www.kaggle.com/c/imagenet-object-localization-challenge/data
